### PR TITLE
Blast legacy 2.2.26

### PR DESCRIPTION
**Pull request by pjbriggs**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This PR adds a recipe for `blast-legacy` version 2.2.26.

### CODE REVIEW

File: recipes/blast-legacy/2.2.26/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cp -R bin $PREFIX
+cp -R data $PREFIX
+cp -R doc $PREFIX
```
Comment (bgruening) [inline recipes/blast-legacy/2.2.26/build.sh]: is doc and data needed?

Comment (pjbriggs) [reply recipes/blast-legacy/2.2.26/build.sh]: These were taken from the `build.sh` from the existing `blast-legacy` 2.2.22 recipe. The doc for standalone `blast` refers to the `data` directory but it needs the user to configure the location by creating a  `.ncbirc` file. I'm happy to remove `data` and `doc`if they're not appropriate in your opinion.

Comment (bgruening) [reply recipes/blast-legacy/2.2.26/build.sh]: I'm just wondering and would like to keep the package small if possible. If there is nothing that is relevant for the binaries, lets remove it.

Comment (pjbriggs) [reply recipes/blast-legacy/2.2.26/build.sh]: No problem, I understand. I'll remove them and if it turns out there's a problem then it's always possible to update in future. Thanks for taking a look!

**Comment by pjbriggs** [comment]
Thanks Bjoern!

**Pull request merged**